charts/ndb-operator/templates/v1_configmap_manager-config.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+data:
+  controller_manager_config.yaml: "apiVersion: controller-runtime.sigs.k8s.io/v1alpha1\nkind:
```

Comment:
let's use `controller_manager_config.yaml: |`
and present properly the content of the file

![image](https://user-images.githubusercontent.com/180613/195771795-b2941f34-19c7-4e45-99e7-dd8909df3506.png)